xtask/src/ci.rs

```diff
@@ -289,17 +289,17 @@ impl CiTask {
         .map_err(Into::into)
     }
 
-    /// Lint ruma-common with clippy with the nightly version and wasm target.
-    ///
-    /// ruma-common is currently the only crate with wasm-specific code. If that changes, this
-    /// method should be updated.
+    /// Lint ruma with clippy with the nightly version and wasm target.
     fn clippy_wasm(&self) -> Result<()> {
         cmd!(
             "
             rustup run {NIGHTLY} cargo clippy --target wasm32-unknown-unknown
-                -p ruma-common --features api,js,rand
+                -p ruma --features __unstable-mscs,api,canonical-json,client-api,events,
```

Comment:
If you are going to insert a line break here, you need to escape it otherwise it will add spaces and break the list:

```suggestion
                -p ruma --features __unstable-mscs,api,canonical-json,client-api,events,\
```

Reply:
ahh i was just debugging this thank you!

Reply:
I had some trouble getting the suggested change to work inside of the `cmd!()` macro. As a stopgap, i just 2 lined it like in other examples.

Reply:
Oh the macro doesn't support escaping newlines? Too bad. This will do.